tests/cephfs/BUG-1980920.py

```diff
@@ -0,0 +1,148 @@
+import json
+import logging
+import random
+import string
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+    try:
+        bz = "1980920"
+
+        fs_util = FsUtils(ceph_cluster)
+
+        log.info("Running cephfs test for bug %s" % bz)
```

Comment:
Kindly avoid blank lines, it is a good practice to use a blank line for logical separation.

Also, opt to use `log.info(f"Running CephFS tests for BZ-{bz}")`

Reply:
I have removed the blank lines
I have changed the "log.info) as well.